src/com/pentalog/pentastagiu/homework2/board/UsersOfBoardApp.java

```diff
@@ -0,0 +1,33 @@
+package com.pentalog.pentastagiu.homework2.board;
+//
+public class UsersOfBoardApp {
+    private String name;
+    private String e_mail;
```

Comment:
nu e nevoie de underscore in numele fieldului, doar la constante (fielduri cu static si final) se foloseste underscore pentru  separare